docs/reference/glossary.md

```diff
@@ -0,0 +1,161 @@
+title: Glossary
+description: Definitions and usage for Filecoin terminology
+---
+
+
+## Address
+
+In the Filecoin network, an _address_ is a unique cryptographic value that serves to publicly identify a user. This value, a public key, is paired with a corresponding private key. The mathematical relationship between the two keys is such that access to the private key allows the creation of a signature that can be verified with the public key. Filecoin specifically employs the Boneh–Lynn–Shacham (BLS) signature scheme for this purpose.
+
+## Block
+
+In a blockchain, a _block_ is the fundamental unit of record. Each block is cryptographically linked to one or more previous blocks. Blocks typically contain [messages](#message) relating changes to some state (for example, financial records) tracked by the blockchain.
+
+## Blockchain
+
+Fundamentally, a _blockchain_ is a system of record in which new records, or [blocks](#block) are cryptographically linked to preceding records. This construction is a foundational component of secure, verifiable, and distributed transaction ledgers.
+
+## Block height
+
+The _height_ of a [block](#block) corresponds to the number of [epochs](#epoch) elapsed before the block was added to the blockchain. The height of the Filecoin [blockchain](#blockchain) is defined to be the maximum height of any block in the blockchain.
+
+## Capacity commitment
+
+If a storage miner doesn't find any available deal proposals appealing, they can alternatively make a _capacity commitment_, filling a [sector](#sector) with arbitrary data, rather than with client data. Maintaining this sector allows the storage miner to provably demonstrate that they are reserving space on behalf of the network.
+
+## Collateral
+
+In order to enter into a [storage deal](#deal), a [storage miner](#storage-miner) is required to provide filecoin as _collateral_, to be paid out as compensation to a client in the event that the miner fails to uphold their storage commitment.
+
+## Deal
+
+Two participants in the Filecoin network can enter into a _deal_ in which one party contracts the services of the other. The Filecoin specification currently details _storage deals_ (in which one party agrees to store data for the other for a specified length of time) and _retrieval deals_ (in which one party agrees to transmit specified data to the other).
+
+## Election
+
+Every [epoch](#epoch), a small subset of Filecoin [storage miners](#storage-miner) are _elected_ to mine a new [block](#block) for the Filecoin blockchain. A miner's probability of being elected is roughly proportional to the share of the Filecoin network's total storage capacity that they contribute.
+
+## Epoch
+
+Time in the Filecoin blockchain is discretized into _epochs_ that are currently twenty-five seconds in length. Each epoch, a subset of storage miners are elected to each add a new block to the Filecoin blockchain via [Winning Proof-of-Spacetime](#winning-proof-of-spacetime-winningpost).
```

Comment:
```suggestion
Time in the Filecoin blockchain is discretized into _epochs_ that are currently twenty-five seconds in length. Every epoch, a subset of storage miners are elected to each add a new block to the Filecoin blockchain via [Winning Proof-of-Spacetime](#winning-proof-of-spacetime-winningpost).
```

Reply:
For what it's worth, per https://www.grammarly.com/blog/everyday-every-day/, I believe both are correct. That said, I have no problem whatsoever changing it.